### ISSUE TITLE
Card didn't show up after home assistant views change

### ISSUE THREAD

**Issue by Sebclem** [dermotduffy/frigate-hass-card#170]
**Checklist:**

- [x] I updated to the latest version available
- [x] I cleared the cache of my browser

**Release with the issue:**

**Last working release (if known):**

**Browser and Operating System:**
- Google Chrome on Windows 10
- Home Assistant companion app on Android

**Description of problem:**

If you change to a different view after loading a first time the view were you have this card, when you come back, the card don't  show up.

Exemple:
My camera View:
![image](https://user-images.githubusercontent.com/23737709/139840875-55c441b6-7896-4fcb-9aaf-3d48d5c663a4.png)

Then I go to my home View:
![image](https://user-images.githubusercontent.com/23737709/139841002-e229e690-c7a5-4801-a2fb-cdb7f56c7eb5.png)

Then back to my Camera View, nothing show up:
![image](https://user-images.githubusercontent.com/23737709/139841079-b7c50165-5b4f-4a80-992e-84da78c3f46c.png)

**Javascript errors shown in the web inspector (if applicable):**

I can't find any relevant message in the debug console.
The only message I found is this one, but it's only a warning and I don't think it's related
```
frigate-hass-card.js?hacstag=394082552120:430 The AudioContext was not allowed to start. It must be resumed (or created) after a user gesture on the page.
```

**Additional information:**
I'm using jsmpeg.

This is my config:
```yml
type: custom:frigate-card
live_provider: frigate-jsmpeg
dimensions:
  aspect_ratio_mode: dynamic
controls: {}
view_default: live
frigate_camera_name: Salon
webrtc: {}
menu_mode: overlay-top
menu_buttons:
  live: true
  clips: true
  frigate: false
```

**Comment by dermotduffy** [comment]
Looks like that's the JSMPEG view you're showing, so I suspect this is the same bug as was fixed in: https://github.com/dermotduffy/frigate-hass-card/pull/123

That was fixed in the pre-release build 1.2.1 that's already available, see this thread for what's in that build and how to install a beta build: https://github.com/blakeblackshear/frigate/discussions/1590#discussioncomment-1491378

Try it and out and see if that fixes it?

**Comment by Sebclem** [comment]
> That was fixed in the pre-release build 1.2.1 that's already available, see this thread for what's in that build and how to install a beta build: blakeblackshear/frigate#1590 (comment)

Confirmed: seems fixed with the beta build 👍 

> PS: Your configuration has many defaults and can probably be simplified to:

Thank !
